cmd/crio/main.go

```diff
@@ -28,6 +27,10 @@ import (
 // It will be populated by the Makefile.
 var gitCommit = ""
 
+const (
+	crioInfoSocket = "/var/run/crio_info.sock"
```

Comment:
Lets move this to /var/run/crio/info.sock
We want to move the crio socket into that directory also,. so lets not make it harder on ourselves

Reply:
fixed Dan